src/observable/EmptyObservable.ts

```diff
@@ -57,11 +57,11 @@ export class EmptyObservable<T> extends Observable<T> {
    * @name empty
    * @owner Observable
    */
-  static create<T>(scheduler?: IScheduler): Observable<T> {
-    return new EmptyObservable<T>(scheduler);
+  static create(scheduler?: IScheduler): Observable<never> {
+    return new EmptyObservable(scheduler);
   }
 
-  static dispatch<T>(arg: DispatchArg<T>) {
+  static dispatch(arg: DispatchArg<never>) {
```

Comment:
it will basically just mean I make `subscriber: Subscriber<never>` instead of `subscriber: Subscriber<T>`